Crashlytics/UnitTests/FIRCLSRecordTests.m

```diff
@@ -214,6 +222,28 @@ - (void)testRecordSignalFile {
   XCTAssertEqual(adapter.storage.total, 63989469184);
 }
 
+- (void)testProtoReport {
+  FIRCLSRecordAdapter *adapter =
+      [[FIRCLSRecordAdapter alloc] initWithPath:[FIRCLSRecordTests persistedCrashFolder]
+                                withGoogleAppId:@"1:17586535263:ios:83778f4dc7e8a26ef794ea"];
+
+  // Used to prototyping:
+  GDTCORTransport *transport = [[GDTCORTransport alloc] initWithMappingID:@"1206"
+                                                             transformers:nil
+                                                                   target:kGDTCORTargetFLL];
+  GDTCOREvent *event = [transport eventForTransport];
+  event.dataObject = adapter;
+  event.qosTier = GDTCOREventQoSFast;  // Bypass batching and have the event get sent out ASAP
+  [transport sendDataEvent:event];
+}
+
+- (void)testProtoReportFromCorruptFiles {
```

Comment:
Something to consider: take a dependency on protobuf in tests and compare the nanopb generated bytes vs. canonical protobuf bytes.

Reply:
I'll add a TODO in this file which will be addressed before the feature branch is merged into master. I'll wait until I have verified that the report is correctly formed and is successfully passed through the pipeline.